Couple of Python test failures:

```
******************** TEST 'Swift(macosx-x86_64) :: Python/build_swift.swift' FAILED ********************
Script:
--
: 'RUN: at line 10';   /Applications/Xcode.app/Contents/Developer/usr/bin/python3 /Users/buildnode/jenkins/workspace/swift-PR-macos-smoke-test/branch-main/swift/utils/build_swift/run_tests.py
--
Exit Code: 1

Command Output (stderr):
--
.....................................................................................................................................................................................................................................................................................................................F.........................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................
======================================================================
FAIL: test_expected_options_exhaustive (tests.build_swift.test_driver_arguments.TestDriverArgumentParser)
Test that we are exhaustively testing all options accepted by the
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/buildnode/jenkins/workspace/swift-PR-macos-smoke-test/branch-main/swift/utils/build_swift/tests/build_swift/test_driver_arguments.py", line 418, in test_expected_options_exhaustive
    self.fail('non-exhaustive expected options, missing: {}'
AssertionError: non-exhaustive expected options, missing: {'--swift-disable-dead-stripping'}

----------------------------------------------------------------------
Ran 1039 tests in 33.278s

FAILED (failures=1)

--

********************
PASS: Swift(macosx-x86_64) :: Evolution/test_enum_add_cases_trap.swift (7486 of 14077)
PASS: Swift(macosx-x86_64) :: Evolution/test_conformance_reference.swift (7487 of 14077)
PASS: Swift(macosx-x86_64) :: Evolution/test_enum_add_cases.swift (7488 of 14077)
FAIL: Swift(macosx-x86_64) :: Python/python_lint.swift (7489 of 14077)
******************** TEST 'Swift(macosx-x86_64) :: Python/python_lint.swift' FAILED ********************
Script:
--
: 'RUN: at line 8';   /Applications/Xcode.app/Contents/Developer/usr/bin/python3 /Users/buildnode/jenkins/workspace/swift-PR-macos-smoke-test/branch-main/swift/utils/python_lint.py
--
Exit Code: 1

Command Output (stdout):
--
./utils/build-script:603:1: W293 blank line contains whitespace
./utils/build_swift/build_swift/driver_arguments.py:337:1: W293 blank line contains whitespace

--

********************
```

--- a/utils/build-script
+++ b/utils/build-script
@@ -600,7 +600,7 @@ class BuildScriptInvocation(object):
             impl_args += ["--skip-build"]
         if not args.build_benchmarks:
             impl_args += ["--skip-build-benchmarks"]
-        
+
         if args.swift_disable_dead_stripping:
             args.extra_cmake_options.append('-DSWIFT_DISABLE_DEAD_STRIPPING:BOOL=TRUE')
 

--- a/utils/build_swift/build_swift/driver_arguments.py
+++ b/utils/build_swift/build_swift/driver_arguments.py
@@ -334,7 +334,7 @@ def create_argument_parser():
            default=defaults.SWIFT_ANALYZE_CODE_COVERAGE,
            help='enable code coverage analysis in Swift (false, not-merged, '
                 'merged).')
-                
+
     option('--swift-disable-dead-stripping', toggle_true, 
            help="Turn off Darwin-specific dead stripping for Swift host tools")
 

--- a/utils/build_swift/tests/build_swift/test_driver_arguments.py
+++ b/utils/build_swift/tests/build_swift/test_driver_arguments.py
@@ -631,6 +631,10 @@ class TestDriverArgumentParser(unittest.TestCase):
         self.assertFalse(namespace.test_android_host)
         self.assertFalse(namespace.build_libparser_only)
 
+    def test_implied_defaults_swift_disable_dead_stripping(self):
+        namespace = self.parse_default_args(['--swift-disable-dead-stripping'])
+        self.assertTrue(namespace.swift_disable_dead_stripping)
+
     def test_build_lib_swiftsyntaxparser_only(self):
         namespace = self.parse_default_args(['--build-libparser-only'])
         self.assertTrue(namespace.build_libparser_only)

--- a/utils/build_swift/tests/expected_options.py
+++ b/utils/build_swift/tests/expected_options.py
@@ -552,6 +552,7 @@ EXPECTED_OPTIONS = [
     EnableOption('--verbose-build'),
     EnableOption('--watchos'),
     EnableOption('--xctest', dest='build_xctest'),
+    EnableOption('--swift-disable-dead-stripping'),
 
     DisableOption('--skip-build-cmark', dest='build_cmark'),
     DisableOption('--skip-build-llvm', dest='build_llvm'),

